README.md

```diff
@@ -1 +1,91 @@
 # meta-xt-prod-cockpit-rcar - Demo product with advanced HMP technologies
+meta-xt-prod-cockpit-rcar
+
+This repository contains Renesas RCAR Gen3-specific Yocto layers for Xen Troops demo and moulin project file to build it. Layers in this repository provide final recipes to build meta-xt-prod-cockpit-rcar demo. This demo is the main Xen Troops product, that we use to develop and integrate new features
+
+Those layers may be added and used manually, but they were written with Moulin build system, as Moulin-based project files provide correct entries in local.conf
+Moulin project file
+
+Features that are present :
+
+    Kingfisher with Starter Kit Premiere 8GB board
+    SD or eMMC boot
+    Android VM support
+
+Building
+Requirements
+
+    Ubuntu 18.0+ or any other Linux distribution which is supported by Poky/OE
+    Development packages for Yocto. Refer to Yocto manual.
+    You need Moulin installed in your PC. Recommended way is to install it for your user only: pip3 install --user git+https://github.com/xen-troops/moulin. Make sure that your PATH environment variable includes ${HOME}/.local/bin.
+    Ninja build system: sudo apt install ninja-build on Ubuntu
+
+Fetching
+
+You can fetch/clone this whole repository, but you actually need only one file from it: prod-cockpit-rcar.yaml. During build moulin will fetch this repository again into yocto/ directory. So, to reduce possible confuse, we recommend to download only prod-cockpit-rcar.yaml:
+
+# curl -O https://raw.githubusercontent.com/xen-troops/meta-xt-prod-cockpit-rcar/master/prod-cockpit-rcar.yaml
```

Comment:
May I ask to format this whlole document using Markdown format? As original README.md was made.
Because right now it is looks horrendous: https://github.com/iusyk/meta-xt-prod-cockpit-rcar/blob/devel-rcar/README.md